src/components/Clock/Clock.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+
+type Props = {
+  name: string
+};
+
+type State = {
+  today: Date
+};
+
+export class Clock extends React.Component<Props, State> {
```

Comment:
The Clock component is not printing a message with the old and new name when the name is changed.